appserver/tests/payara-samples/samples-programmatic/pom.xml

```diff
@@ -19,6 +19,10 @@
             <groupId>fish.payara.extras</groupId>
             <artifactId>payara-micro</artifactId>
         </dependency>
+        <dependency>
+            <groupId>fish.payara.server.internal.extras</groupId>
+            <artifactId>payara-micro-boot</artifactId>
+        </dependency>
```

Comment:
contents of micro boot is within `payara-micro`. Why is explicit dependency to internal module necessary?

Reply:
Not true. Micro's dependencies are all optional, so they have to be explicitly declared.